packages/multi-select-combo-box/theme/material/vaadin-multi-select-combo-box.js

```diff
@@ -3,7 +3,6 @@
  * Copyright (c) 2021 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import '@vaadin/overlay/theme/material/vaadin-overlay.js';
```

Comment:
Same as above, this import it's no longer needed as the component does not define `vaadin-overlay`.